Documentation/contributing/development/reviewers_committers/review_docs.rst

```diff
@@ -0,0 +1,240 @@
+documentation. Each time a non-draft PR touching files owned by the team is
+opened on GitHub, one member of the team is automatically assigned for review.
+
+To join the team, you must be a Cilium Reviewer. See `Cilium's Contributor
+Ladder`_ for details on the requirements and the
+application process.
+
+.. _docs-structure_team: https://github.com/orgs/cilium/teams/docs-structure
+.. _Cilium's Contributor Ladder: https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md
+
+Reviewing Pull Requests
+=======================
+
+This section describes some of the process and expectations for reviewing PRs
+on behalf of cilium/docs-structure. Note that :ref:`the generic PR review
+process for Committers <review_process>` applies, even though it is not
+specific to documentation.
+
+Technical Contents
+------------------
+
+You are not expected to review the technical aspects of the documentation
+changes in a PR, however, if you do have knowledge of the topic and if you find
+some elements that are incorrect or missing, do flag them.
+
+Documentation Structure
+-----------------------
+
+One essential part of a review is to ensure that the contribution maintains a
+coherent structure for the documentation. Ask yourself if the changes are
+located on the right page, at the right place. This is especially important if
+pages are added, removed, or shuffled around. If the addition is large,
+consider whether the page should be split. Consider also whether new text comes
+with a satisfactory structure. For example, does it fit well with the
+surrounding context, or did the author simply use a "note" box instead of
+trying to integrate the new information to the relevant paragraph?
+
+See also :ref:`the recommendations on documentation structure for contributors
+<docs_structure_recommendations>`.
+
+Specific Items to Look Out For
+------------------------------
+
+Backports Labels
+~~~~~~~~~~~~~~~~
+
+See :ref:`the backport criteria for documentation changes
+<backport_criteria_docs>`. Mark the PR for backports by setting the labels for
+all supported branches to which the changes apply, that is to say, all
+supported branches containing the parent features to which the modified
+sections relate.
+
+CODEOWNERS Updates
+~~~~~~~~~~~~~~~~~~
+
+All documentation sources are assigned to cilium/docs-structure for review by
+default. However, when a contributor creates a new page, consider whether it
+should be covered by another team as well so that this other team can review
+the technical aspects. If this is the case, ask the author to update the
+CODEOWNERS file.
+
+Beta Disclaimer
+~~~~~~~~~~~~~~~
+
+When a feature is advertised as Beta in the PR, make sure that the author
+clearly indicates it as such in the documentation, both by mentioning "(Beta)"
+in the title of the section for the feature and by including the dedicated
+banner. Users can include this banner by using the ``include`` directive for
+the ``Documentation/beta.rst`` fragment.
```

Comment:
It would be good to include an example here:
```suggestion
the ``Documentation/beta.rst`` fragment.

.. parsed-literal::
    
    .. include:: Documentation/beta.rst
```

Reply:
I didn't want initially because the relative path depends on where we include from, and I didn't want to mislead the readers; but it's true we can just as well use an absolute path.